.golangci.yml

```diff
@@ -1,204 +1,173 @@
+        - pattern: fmt\.Print.*(# Disallowed function used\. Use environments functions for printing or to a specific writer from environment\.InputOutput\(\)\.)?
+        - pattern: os\.Stdout(# Disallowed output streams used\. Use environment\.InputOutput\(\).Out instead\.)?
+        - pattern: os\.Stderr(# Disallowed output streams used\. Use environment\.InputOutput\(\).Out instead\.)?
+        - pattern: os\.Stdin(# Disallowed output streams used\. Use environment\.InputOutput\(\).Out instead\.)?
+    gocritic:
+      disabled-checks:
+        - appendAssign
+        - argOrder
+        - assignOp
+        - captLocal
+        - dupArg
+        - elseif
+        - exitAfterDefer
+        - ifElseChain
+        - mapKey
+        - singleCaseSwitch
+        - unlambda
+        - wrapperFunc
+    gosec:
+      includes:
+        - G101
+        - G102
+        - G103
+        - G104
+        - G106
+        - G108
+        - G109
+        - G111
+        - G113
+        - G201
+        - G202
+        - G203
+        - G303
+        - G307
+        - G403
+        - G502
+        - G503
+        - G504
+        - G601
+    govet:
+      disable:
+        - shadow
+        - fieldalignment
+      enable-all: true
+      settings:
+        printf:
+          funcs:
+            - Print
+            - Printf
+            - Println
+            - Debug
+            - Debugf
+            - Info
+            - Infof
+            - Warn
+            - Warnf
+            - Error
+            - Errorf
+            - github.com/stackrox/rox/migrator/log.WritetoStderr
+            - github.com/stackrox/rox/migrator/log.WritetoStderrf
+    nolintlint:
+      require-explanation: false
+      require-specific: true
+      allow-unused: false
+    revive:
+      rules:
+        - name: package-comments
+          disabled: true
+        - name: error-strings
+          disabled: true
+        - name: unexported-return
+          disabled: true
+    staticcheck:
+      checks: [all,-ST1000,-ST1001,-ST1003,-ST1005,-SA1019,-SA4001,-ST1016]
+    wrapcheck:
+      ignore-sig-regexps:
+        - utils\.Should
+        - errox\..+\.CausedBy(f)?
+        - retry\.MakeRetryable
+        - policy\.NewErr.*
+  exclusions:
+    generated: lax
+    rules:
+      - linters:
+          - wrapcheck
+        path: (central|compliance|integration-tests|local|migrator|operator|pkg|scanner|sensor|tests|tools|scale)/
+      - linters:
+          - forbidigo
+        path: (central/graphql/schema/print|compliance|integration-tests|local|migrator|operator|pkg|scanner|sensor|tests|tools|scale|govulncheck)/
+      - linters:
+          - forbidigo
+        path: roxctl/central/generate/interactive.go
+      - linters:
+          - forbidigo
+          - wrapcheck
+        path: _test\.go
+      - linters:
+          - forbidigo
+        path: roxctl/common/io/io\.go
+      - linters:
+          - staticcheck
+        path: .*/bleve@.*/index\.go
+        text: SA4023
```

Comment:
For follow-up: do we still use bleve ? If not, this can probably be removed. Otherwise the comment (origin line 44) should be ported here.

Reply:
Bleve is not used anymore in the codebase, although there remain a few references in code comments.